src/docs/self-hosted/index.mdx

```diff
@@ -36,7 +35,7 @@ Environment="NO_PROXY=127.0.0.0/8"
    {
      "httpProxy": "http://proxy:3128",
      "httpsProxy": "http://proxy:3128",
-     "noProxy": "127.0.0.0/8"
+     "noProxy": "relay,web,sentry,127.0.0.0/8"
```

Comment:
Have you tested a clean working installation with these variables? Are these enough? We have `kafka:9092` in `relay/config.yml` for example, shouldn't we also add `kafka` to `noProxy`?

Reply:
I'd removed all docker containers, images, volumes and networks and reinstalled Sentry one more time, checked web, created test event. I'd found no errors in docker logs.

Reply:
Please wait a bit, I need to recheck it one more time.

Reply:
From the one hand proxies from ~/.docker/config.json work only for docker run command and do not work for docker build. It is written in the [docker docs](https://docs.docker.com/network/proxy/):

> When you create or start new containers, the environment variables are set automatically within the container.

From the other hand we need a connection to the Internet only during build stage. So we do not need `~/.docker/config.json` at all. We need to set correct env variables through `/etc/environment` and that's it. In this commit https://github.com/getsentry/self-hosted/commit/2672ce545227f9a40b90178006c5ab25f7860a64 we do not need changes in `/etc/apt/apt.conf`, because all required variables will be passed through `/etc/environment`.

Sorry for my mistake.

Reply:
@aminvakil  I'd updated PR, please take a look.